src/components/webviews/jsInteractions/jsCozyInjection.ts

```diff
@@ -21,13 +21,16 @@ const makeMetadata = routeName => {
   })
 }
 
-export const jsCozyGlobal = (routeName, isSecureProtocol) => `
+export const jsCozyGlobal = (
+  routeName?: string,
+  isSecureProtocol?: boolean
+): string => `
   if (!window.cozy) window.cozy = {}
   window.cozy.isFlagshipApp = true
   window.cozy.ClientKonnectorLauncher = 'react-native'
   window.cozy.ClientConnectorLauncher = 'react-native' // deprecated
-  window.cozy.flagship = ${makeMetadata(routeName)}
-  window.cozy.isSecureProtocol = ${isSecureProtocol || 'false'}
+  window.cozy.flagship = ${routeName ? makeMetadata(routeName) : ''}
+  window.cozy.isSecureProtocol = ${isSecureProtocol ? 'true' : 'false'}
```

Comment:
This `isSecureProtocol ? 'true' : 'false'` seems weird

Reply:
I agree but TS doesn't like outputting non-string values in template literals in this case